platform/core/PolyPodCoreSwift/Sources/PolyPodCoreSwift/MessagePack+Decoding.swift

```diff
@@ -41,6 +41,25 @@ extension MessagePackValue {
             throw DecodingError.invalidValue(info: "Expected string, received \(self)")
         }
     }
+
+    func getBool() throws -> Bool {
+        if let bool = self.boolValue {
+            return bool
+        } else {
+            throw DecodingError.invalidValue(info: "Expected bool, received \(self)")
+        }
+    }
+    
+    func getUInt() throws -> UInt? {
+        guard self != nil else {
+            return nil
+        }
+        if let uint = self.uintValue {
+            return uint
+        } else {
+            throw DecodingError.invalidValue(info: "Expected uint, received \(self)")
+        }
```

Comment:
would prefer the `throw` first here so we avoid the else
```suggestion
        if uint != self.uintValue {
            throw DecodingError.invalidValue(info: "Expected uint, received \(self)")
        } 
        return uuint;
```

Reply:
I don't understand where `uint` variable is assigned in your suggestion.

Reply:
Probably you wan to suggest a `guard let` statement.